### PR TITLE
Issue3139 mz controller without changes to pressure independent damper option

### PR DESCRIPTION
**Pull request by JayHuLBL**
This address the issues in #3139, except the last two items about the pressure independent damper.

### CODE REVIEW

File: Buildings/Examples/VAVReheat/BaseClasses/VAVReheatBox.mo [hunk 1]
```diff
@@ -72,7 +72,7 @@ model VAVReheatBox "Supply box of a VAV system with a hot water reheat coil"
     "Supply Air Volumetric Flow Rate"
     annotation (Placement(transformation(extent={{100,70},{120,90}}),
         iconTransformation(extent={{100,70},{120,90}})));
-  Buildings.Fluid.Actuators.Dampers.PressureIndependent vav(
+  Buildings.Fluid.Actuators.Dampers.Exponential vav(
```
Comment (AntoineGautier) [inline Buildings/Examples/VAVReheat/BaseClasses/VAVReheatBox.mo:75]: If we don't address the issue with `have_preIndDam` in this PR then this modification must be undone.
Note that

- The zone temperature setpoint is no longer met in the examples with this change.
- `Buildings.Examples.VAVReheat.BaseClasses.Controls.RoomVAV` which is used for the ASHRAE2006 example does not include a pressure independent *control logic*, so `Dampers.PressureIndependent` is needed there.
- However, I noticed that on the master branch, `have_preIndDam` is left at its default value `false` for the Guideline36 example, despite the model using `Dampers.PressureIndependent`. This is a mistake that should be corrected (or that may become obsolete after refactoring to remove `have_preIndDam` completely). But again, not through this PR I believe.

**Review by AntoineGautier — CHANGES_REQUESTED** [review]
See my inline comment.
In 72e208fbfc I also modified the `FreezeStat` enumeration as the previous description strings were not clear enough (as discussed yesterday).
- [x] Some reference results may still need to be updated (after undoing the change of the damper model used in the example models).

File: Buildings/Examples/VAVReheat/BaseClasses/VAVReheatBox.mo (hunk 1 above)
Comment (JayHuLBL) [reply Buildings/Examples/VAVReheat/BaseClasses/VAVReheatBox.mo]: @AntoineGautier Thanks for catching it. I have reverted the changes and Rerun the unit test. There is no reference results change.

**Review by AntoineGautier — APPROVED** [review]

**Comment by JayHuLBL** [comment]
@mwetter Can you merge this PR if you are fine with the changes? I have no access to merge PR.

**Pull request merged**